Assets/Scripts/API/Save/SaveTreeHeader.cs

```diff
@@ -25,15 +25,17 @@ namespace DaggerfallConnect.Save
     public class SaveTreeHeader
     {
         // Consts
-        public const int HeaderLength = 18;
-        public const int VersionNumber = 0x26;
+        public const int HeaderLength = 19;
+        public const int VersionNumberNew = 0x126; // Save from Daggerfall 2.17
```

Comment:
Well actually since it will just be one version number, as before, I'll remove the comment. I wonder what version number saves from Daggerfall versions between 1.07.213 and 1.0 were.